app/lib/features/chat/widgets/convo_card.dart

```diff
@@ -84,7 +84,9 @@ class ConvoCard extends ConsumerWidget {
                         overflow: TextOverflow.ellipsis,
                       ),
                 subtitle: buildSubtitle(context, constraints),
-                trailing: constraints.maxWidth < 300 ? null : trailing,
+                trailing: constraints.maxWidth < 300
```

Comment:
Didn't getting visual idea about issue point and resolution by this code change.

Can you elaborate more about this in PR Description?

Reply:
Added description.

Reply:
what is difference between `trailing`  & `_TrailingWidget(roomId: roomId)` ?

Reply:
It serves as fallback if ```trailing``` param isn't provided.